src/main/java/racingcar/Car.java

```diff
@@ -3,13 +3,23 @@
 
 public class Car {
     private static final int MOVE_MIN_NUM = 4;
+    private final String carName;
 
-    private int position;
+    private int position = 0;
 
-    public Car(int position) {
+    public Car(String carName) {
+        this.carName = carName;
```

Comment:
다른 생성자를 호출하는 방식으로 변경해보면 어떨까요?
```suggestion
        this(carName, 0);
```